.prettierignore

```diff
@@ -19,4 +19,7 @@ languages/fsharp/exercises/**/obj/
 languages/javascript/exercises/**/node_modules/
 
 # TypeScript-specific files
-languages/typescript/exercises/**/node_modules/
\ No newline at end of file
+languages/typescript/exercises/**/node_modules/
+
+# Julia-specific files
+languages/julia/exercises/**/.meta/config.json
```

Comment:
These are semi-generated, this will allow us to use the default JSON.jl formatting without prettier complaining about it.